newscoop.conf

```diff
@@ -0,0 +1,13 @@
+<VirtualHost *:80>
+      DocumentRoot /var/www/newscoop
+      ServerName docker.newscoop
```

Comment:
btw, should note that vhost is not required on user's machine, but rather /etc/hosts entry. but again, not sure how resolve without requiring this for mac os users.